components/support/nimbus-fml/src/error.rs

```diff
@@ -25,8 +25,8 @@ pub enum FMLError {
     #[allow(dead_code)]
     #[error("Internal error: {0}")]
     InternalError(&'static str),
-    #[error("Validation Error: {0}")]
-    ValidationError(String),
+    #[error("Validation Error at {0}: {1}")]
+    ValidationError(String, String),
```

Comment:
🤩 I was wrestling with this code a while ago to see if I can pull line numbers, but that sounded like a **very** big pain. I'm still hoping that one day we can do that (probably means not using serde_yaml 😞)

But regardless thank you for this!!